src/include/misc.h

```diff
@@ -249,6 +249,18 @@
 	(dst).size = (src).size;					\
 } while (0)
 
+/* Timestamp type and helper macros. */
+#ifdef HAVE_TIMESTAMPS
+#define	WT_TIMESTAMP(x) (x)
+typedef uint8_t wt_timestamp_t[TIMESTAMP_SIZE];
+#define	WT_DECL_TIMESTAMP(x) wt_timestamp_t x;
```

Comment:
Regarding the semicolon: I couldn't figure out a way to move it outside the macro without breaking the no-timestamp case.  Happy to take suggestions.